### PR TITLE
seport: minor refactor

### PR DESCRIPTION
**Pull request by russoz**
##### SUMMARY
Minor refactor

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME
plugins/modules/system/seport.py

### CODE REVIEW

File: plugins/modules/system/seport.py [hunk 1]
```diff
@@ -113,7 +113,9 @@
 
 
 def get_runtime_status(ignore_selinux_state=False):
-    return True if ignore_selinux_state is True else selinux.is_selinux_enabled()
+    if ignore_selinux_state is False:
+        return selinux.is_selinux_enabled()
+    return True
```
Comment (felixfontein) [inline plugins/modules/system/seport.py:118]: I would do it the other way around:
```suggestion
    if ignore_selinux_state is True:
        return True
    return selinux.is_selinux_enabled()
```
Quick exit if `ignore_selinux_state` is `True`, and the 'proper' code as the main part of the function. (Not that the proper code here is much longer than the quick exit...)

**Review by felixfontein — COMMENTED** [review]
Besides this, LGTM.

File: plugins/modules/system/seport.py (hunk 1 above)
Comment (russoz) [reply plugins/modules/system/seport.py:118]: Kinda personal preference, I suppose. I am bothered by code like:
```python
if x is True:
    return True
```
In terms of compilation, both will likely produce the exact same low level code.

Comment (felixfontein) [reply plugins/modules/system/seport.py]: You can also change it to `if x: return True` :-) Or change the whole function body to `return ignore_selinux_state or selinux.is_selinux_enabled()` ;)

Comment (russoz) [reply plugins/modules/system/seport.py]: I like that.

File: plugins/modules/system/seport.py [hunk 2]
```diff
@@ -113,7 +113,9 @@
 
 
 def get_runtime_status(ignore_selinux_state=False):
-    return True if ignore_selinux_state is True else selinux.is_selinux_enabled()
+    if ignore_selinux_state is True:
+        return True
+    return selinux.is_selinux_enabled()
```
Comment (russoz) [inline plugins/modules/system/seport.py:118]:
```suggestion
    return ignore_selinux_state or selinux.is_selinux_enabled()
```

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@russoz thanks for improving this module!

**Pull request merged**